src/components/NotFound/NotFound.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 
-const NotFound = () => (<h1>NotFound</h1>);
+const NotFound = () => (<h1>404 NotFound</h1>);
```

Comment:
You can omit braces. When jsx is one line it doesn't need to be surrounded by braces